### DISCUSSION TITLE
Get value from table

### DISCUSSION THREAD

**Discussion by TheStelarek** [UI5/webcomponents#7417] [Q&A]
Hello,

1. How can I get the value from selected row in table?
2. Is there a any way to add a pagination to the table?

Greetings!

**Comment by Qcyk** [comment]
Hi, I have the same problem. How to get a element from selected row? Is there a any plan to add method to get it? Or only one solution is find it in HTML tree?

**Comment by Lukas742** [comment]
Hi,

> 1. How can I get the value from selected row in table?

You can get the selected rows inside the `onSelectionChange` event, by accessing the detail property. There you can compare the `selectedRows` array with the `previouslySelectedRows` array, to receive the row where the selection has been changed.
For the `onRowClick` event it's similar, but there only the clicked row is returned.
If you'd like to pass custom data to the event, you can use this [approach](https://sap.github.io/ui5-webcomponents-react/?path=/docs/knowledge-base-ids-via-dataset--docs).

> 2. Is there a any way to add a pagination to the table?

There is an open discussion about general pagination, you can find it [here](https://github.com/SAP/ui5-webcomponents/issues/6162).

___ 

Since the `Table` is a web component developed by the ui5-webcomponents team, I'm going to forward this Discussion to their repo, so they can add information if I've missed something.
